### PR TITLE
update logo: "ZF2" to general "ZF" and update wording of ZF2 -> ZF3 and various link fixes

### PR DESCRIPTION
**Pull request by samsonasik**
in favor of https://github.com/zendframework/ZendSkeletonApplication/pull/329#issuecomment-213469920

### CODE REVIEW

**Comment by samsonasik** [comment]
@weierophinney  I've updated ZF2 -> ZF3 wordings and various link fixes as well ;)

**Comment by armenio** [comment]
would be interesting to use a var/const for version label? - keep in mind that this will change again in the future

**Comment by samsonasik** [comment]
the version usage already in https://github.com/samsonasik/ZendSkeletonApplication/blob/150ef951e8c2f8d6134fc163cea2714c367c5ce5/module/Application/view/application/index/index.phtml#L3 for specific X.Y.Z version, other than it is a big major one, we may have different template for ZF4 imo :)

File: module/Application/view/application/index/index.phtml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="jumbotron">
-    <h1><?php echo sprintf('Welcome to %sZend Framework 2%s', '<span class="zf-green">', '</span>') ?></h1>
-    <p><?php echo sprintf('Congratulations! You have successfully installed the %sZF2 Skeleton Application%s. You are currently running Zend Framework version %s. This skeleton can serve as a simple starting point for you to begin building your application on ZF2.', '<a href="https://github.com/zendframework/ZendSkeletonApplication" target="_blank">', '</a>', \Application\Module::VERSION) ?></p>
-    <p><a class="btn btn-success btn-lg" href="https://github.com/zendframework/zf2" target="_blank">Fork Zend Framework 2 on GitHub &raquo;</a></p>
+    <h1><?php echo sprintf('Welcome to %sZend Framework 3%s', '<span class="zf-green">', '</span>') ?></h1>
```
Comment (weierophinney) [inline module/Application/view/application/index/index.phtml]: One thing we're working on with GitHub is renaming the repository from "zf2" to "zendframework". As such, I'd like to actually *remove* all references to the version, other than the line indicating which version is currently installed.

I'll make those changes on merge.

**Pull request merged**